synthesizer/src/store/transaction/deployment.rs

```diff
@@ -56,6 +56,9 @@ pub trait DeploymentStorage<N: Network>: Clone + Send + Sync {
     type FeeMap: for<'a> Map<'a, N::TransactionID, (N::TransitionID, N::StateRoot, Option<Proof<N>>)>;
     /// The mapping of `fee transition ID` to `transaction ID`.
     type ReverseFeeMap: for<'a> Map<'a, N::TransitionID, N::TransactionID>;
+    /// The mapping of `transaction ID` to `Admin`.
+    type AdminMap: for<'a> Map<'a, N::TransactionID, Admin<N>>;
```

Comment:
Discussed offline. We'll change this layout to `(program ID, edition) => Admin<N>`.

Reply:
Done, though we may want to consider program upgrades and edition bumps.